modules/utils/src/logOperationTime.ts

```diff
@@ -0,0 +1,33 @@
+import pino from "pino";
+import { performance } from 'perf_hooks';
+import { getRandomBytes32 } from "@connext/vector-utils";
+
+/// This method is used to log the start and end time of an executed method. We are using the
+/// perf_hooks (performance) lib provided by node to calculate this - much more accurate than
+/// Date.now(). On first call, a unique label string is generated and returned. On second call,
+/// same label string should be passed in to measure and log the diff in time (i.e. amount of
+/// time method took to execute in ms).
+export const logOperationTime = (log: pino.BaseLogger, method: string, label?: string): string => {
+  if (label) {
+    const stopLabel = `${label}-stop`;

```

Comment:
What would happen if there are label collisions?

Reply:
Currently using a unique identifier below as part of generated label to prevent that, but this will be replaced by passed in methodId